docs/en/advanced-concepts/migrating-to-appium-2.0.md

```diff
@@ -163,7 +163,30 @@ TODO
 
 #### :tada: Driver and Plugin CLI args
 
-TODO
+#### CLI Argument Migrations
```

Comment:
I'd say all these almost all args should be removed from v2 as they are:
- driver-specific
- could be replaced with default capabilities
It only makes sense to keep these, that a system admin would potentially like to keep, in order to override user-provided capabilities on the server side, for which we could have potentially have another common argument. Something like `enforceCapabilities`. Capabilities provided there would have priority over all other values included these provided by users